packages/client/e2e/pages/FxNewRfqPageObject.ts

```diff
@@ -0,0 +1,31 @@
+import { Page, TestInfo } from "@playwright/test"
+
+export class FxNewRfqPageObject {
+  constructor(
+    readonly page: Page,
+    readonly testInfo: TestInfo,
+  ) {}
+
+  async clickCurrency(currency: string) {
+    return await this.page
+      .getByTestId("tab-bar-tabs")
+      .getByText(currency)
+      .click()
+  }
+
+  get spotTileNotionalInput() {
+    return this.page.locator("input[id='notional-input-EURUSD']")
```

Comment:
this includes a specific currency pair
pass this in perhaps as a `currencyPair` param?

Reply:
as you have done below .. but please use `currencyPair`, no need to abbreviate to `pair`

Reply:
done